reference-implementation/lib/transform-stream.js

```diff
@@ -379,23 +435,26 @@ class TransformStream {
 
     this._transformStreamController = new TransformStreamDefaultController(this);
 
+    this._readableType = transformer.readableType;
```

Comment:
I was relying on the other two stream constructors already validating it, but I suppose not.